### PR TITLE
LOOM-1292 BpkInfoBannerInner - Class 2 Overrides 

### PR DESCRIPTION
**Pull request by jronald01**
**Class 2 Overrides:**

1. BpkIcon - Styles added to a wrapper and Down Chevron icon added to avoid misalignment that comes from adding a wrapper which essentially takes the height from the parent and causes a misalignment in one direction when you apply `scaleY(-1)`.
2. BpkCloseButton - Duplicated styles removed
3. BpkLink - Style is redundant

Remember to include the following changes:

- [x] Ensure the PR title includes the name of the component you are changing so it's clear in the release notes for consumers of the changes in the version e.g `[KOA-123][BpkButton] Updating the colour`
- [ ] `README.md` (If you have created a new component)
- [ ] Component `README.md`
- [ ] Tests
- [ ] Storybook examples created/updated
- [ ] Type declaration (`.d.ts`) files updated
- [ ] For breaking changes or deprecating components/properties, migration guides added to the description of the PR. If the guide has large changes, consider creating a new Markdown page inside the component's docs folder and link it here

### CODE REVIEW

**Comment by jronald01** [comment]
After playing around and trying to fix the alignment, I’ve decided it’s simpler just to pull in the down chevron icon and make the icon conditional.

The problem was that once we introduced the new wrapper and flipped the icon `scaleY (-1)`, we are now flipping an area that is the same height as the `button` rather than the `svg`, hence one of the directions up or down would no longer be aligned due to flipping a different sized area, for example, at 4x zoom, we are now flipping a 39.06x56.15 rather than the original 39.06x46.39 which causes a misalignment.

File: packages/bpk-component-info-banner/src/BpkInfoBannerInner.tsx [hunk 1]
```diff
@@ -56,6 +58,7 @@ import STYLES from './BpkInfoBanner.module.scss';
 const getClassName = cssModules(STYLES);
 
 const ExpandIcon = withButtonAlignment(ChevronDownIcon);
+const ContractIcon = withButtonAlignment(ChevronUpIcon);
```
Comment (olliecurtis) [inline packages/bpk-component-info-banner/src/BpkInfoBannerInner.tsx:61]: It took me a while to understand what `Contract` meant here, maybe better and what we have used in other components would be `Collapse` so keeps it consistent across Backpack, so would be 

```suggestion
const CollapseIcon = withButtonAlignment(ChevronUpIcon);
```

File: packages/bpk-component-info-banner/src/BpkInfoBannerInner.tsx [hunk 2]
```diff
@@ -107,7 +107,9 @@ const ToggleButton = (props: ToggleButtonProps) => {
       aria-expanded={props.expanded}
       title={props.label}
     >
-      <ExpandIcon className={classNames} />
+      <div className={classNames}>
+        {props.expanded ? <ExpandIcon/> : <ContractIcon/>}
```
Comment (olliecurtis) [inline packages/bpk-component-info-banner/src/BpkInfoBannerInner.tsx:111]: With the above suggestion
```suggestion
        {props.expanded ? <ExpandIcon/> : <CollapseIcon/>}
```

**Review by olliecurtis — CHANGES_REQUESTED** [review]

File: packages/bpk-component-info-banner/src/BpkInfoBannerInner.tsx (hunk 1 above)
Comment (jronald01) [reply packages/bpk-component-info-banner/src/BpkInfoBannerInner.tsx:61]: Ah, yeah, I was thinking expansion and contraction, but yeah, **collapse** makes way more sense 👍

**Comment by jronald01** [comment]
Reversed chevron direction by a mistake when converting to a conditional, fixed it now:
`{props.expanded ? <CollapseIcon/> : <ExpandIcon/> }`

i.e. when it's expanded, we want to show the **Collapse Icon** and when it's not expanded, we want to show the **Expand Icon**.

**Review by olliecurtis — APPROVED** [review]

**Pull request merged**